mailpoet/lib/Automation/Integrations/MailPoet/Templates/WorkflowBuilder.php

```diff
@@ -58,7 +58,7 @@ public function welcomeEmailSequence(string $name): Workflow {
     $sendFirstEmailStep = $this->sendEmailActionStep();
     $firstDelayStep->setNextStepId($sendFirstEmailStep->getId());
 
-    $secondDelayStep = $this->delayStep( null,"HOURS");
+    $secondDelayStep = $this->delayStep( null, "HOURS");
```

Comment:
Only pointing this out because there are merge conflicts anyway that need to be resolved: there's still a leading space before `null` and we could use single quotes for the string. Same on line 55.

Reply:
Why isn't this catched by our coding standards?

Reply:
I was wondering the same thing myself! Maybe there's a PhpStan rule we should add? I don't know too much about PhpStan myself and after a little bit of googling I was no closer to figuring out how we would add a rule to catch this.